e2e/scripts/st_arrow_line_chart_add_rows_special.py

```diff
@@ -0,0 +1,18 @@
+import time
```

Comment:
Can you add a comment explaining what this is testing? (Is this a test that would have failed prior to the PR, and is therefore testing something about the immutability of add rows?)